scripts/azure-pipelines.yml

```diff
@@ -311,5 +311,29 @@ jobs:
       ./scripts/install_system_deps.sh  # no intel python
       ./scripts/install_python_deps.sh  # numpy, conda-build and etc.
       echo ========================= build DPNP package ===============================
-      export DPNP_BACKEND_TESTS_ENABLE=1
       ./0.build.sh
+
+# DPNP backend clib tests building and running
+- job: test_dpnp_backend_clib_lin
+  displayName: test_dpnp_backend_clib_lin
+  pool:
+    vmImage: 'ubuntu-20.04'
+  steps:
+  - bash: |
+      . ./scripts/install_cmake_lin.sh
+      ./scripts/install_system_deps.sh  # no intel python
+      echo ========================= build DPNP package ===============================
+      export DPNP_BACKEND_TESTS_ENABLE=1
+      . ./0.env.sh
+      THEDIR=$(pwd)
+      cmake  -S$THEDIR/dpnp/backend -B$THEDIR/build/ -DCMAKE_BUILD_TYPE=Release \
+              -DDPNP_INSTALL_PREFIX=$THEDIR/dpnp -DDPNP_INSTALL_STRUCTURED=OFF \
+              -DDPNP_SYCL_QUEUE_MGR_ENABLE:BOOL=OFF \
+              -DDPNP_QUEUEMGR_INCLUDE_DIR=No_sycl_queue_mgr_include_dir \
+              -DDPNP_QUEUEMGR_LIB_DIR=No_sycl_queue_mgr_library_dir \
+              -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -DDPNP_BACKEND_TESTS:BOOL=ON \
```

Comment:
I would propose to do not use all these variables here (like `DPNP_QUEUEMGR_INCLUDE_DIR`). Most of these variables have default values that should be used here (just do not put them here).